config.yaml

```diff
@@ -17,9 +17,11 @@ password:
 database:
 # Endpoint for scraping prometheus metrics. Default localhost:9090.
 metrics_endpoint: localhost:9090
-# Table with spans. Default "jaeger_spans_local".
+# Whether to use sql scripts supporting replication and sharding. Default false.
+replication:
+# Table with spans. Default "jaeger_spans".
 spans_table:
-# Span index table. Default "jaeger_index_local".
```

Comment:
Why are we changing the names? 

The default config should use local tables. I was thinking that if replication is enabled the default names would be changed to `jaeger_spans` if they are not provided by the user.

Reply:
I'm adding suffix to local tables automatically, but it really isn't clear from comments. Could you please give me some advice?

Reply:
e.g. 
```
# Span index table. Default "jaeger_index_local", when replication is enabled default is `jaeger_index`.
```